docs/website/versioned_docs/version-3.0.0/command-reference/delete.md

```diff
@@ -1,40 +1,39 @@
 ---
-title: odo delete
+title: odo delete component
 sidebar_position: 4
 ---
 
-`odo delete` command is useful for deleting resources that are managed by odo.
+`odo delete component` command is useful for deleting resources that are managed by odo. It deletes the component and it's related innerloop and outerloop resources from the cluster.
 
-## Deleting a component
-
-To delete a _devfile_ component, you can execute `odo delete`.
+There are 2 ways to deleting a component:
+1. [Delete with access to Devfile](/#Delete with access to Devfile)
+2. [Delete without access to Devfile](/#Delete without access to Devfile)
 
+## Delete with access to Devfile
 ```shell
-odo delete
+odo delete component
 ```
+odo analyzes the devfile for the devfile component and the outerloop resources.
+If the component has been deployed on the cluster, then odo will list all the resources and prompt the user to confirm the deletion.
+Otherwise, odo will exit with a message stating that it could not find the resources on the cluster.
```

Comment:
* It's worth mentioning here that one needs to `cd` into a component directory for this command to work successfully. If they don't have a component directory for any reason, they could look at the section about deletion without access to devfile.
* > odo analyzes the devfile for the devfile component

  IMO, it's an odo component, not a devfile component